docs/development/renovate-tutorial.md

```diff
@@ -0,0 +1,134 @@
+# Renovate Hands-on-tutorial
+
+##Introduction
+We will learn how to run renovate and experience how it works.
+There are Multiple ways to run Renovate, using CLI, self-hosted server or docker,
+but the easiest way to start with is the [Renovate App](https://github.com/apps/renovate).
+
+You can trigger renovate manually from CLI or docker by running it to pick up any changes needed,
+or you can run [Renovate App](https://github.com/apps/renovate) or a host renovate on a server,
+and it will pick up any needed changes every few minutes/hours.
+
+## Tutorial
+
+- We will work with Github for this tutorial since our app is hosted on Github, open your Github repository
+- Fork [this](https://github.com/PhilipAbed/RenovateTutorial) repository
```

Comment:
Hi @rarkins , I'm waiting for a renovate tutorial repository instead of this URL where we can add to it. i hope we aren't using my own repository for tutorial :D

Reply:
I think this file should be located in that repo too, but we can continue reviewing this doc here for now